README.md

```diff
@@ -29,7 +29,8 @@ Body:
 
 ```typescript
 {
-  success: Boolean
+  success: Boolean,
+  walletAddress: String
```

Comment:
Uh oh, when reviewing only this change, I find it confusing to see `success` in the request body. This field is usually returned by the server, so I had to double check that this part of our docs is describing the request body.

How do you feel about renaming `success` to something more descriptive, e.g. `retrievalSucceeded`?

I am fine to leave that out of this PR, just pointing out my surprise.

Reply:
> Uh oh, when reviewing only this change, I find it confusing to see success in the request body. This field is usually returned by the server, so I had to double check that this part of our docs is describing the request body.

Interesting, I hadn't thought about it this way at all. I guess to me `success` is _not_ a typical response field, because you should use HTTP code 200 to mark a success.

I was halfway through changing it to `retrievalSucceeded`, but then realized an issue: If you call a field "retrievalSucceeded' when updating a retrieval, you must prefix all fields with "retrieval". Otherwise, you get inconsistent naming. I'm therefore going to keep it as is for now, let's reevaluate in the future.